app/src/main/java/com/nervousfish/nervousfish/modules/pairing/AndroidNFCAdapter.java

```diff
@@ -0,0 +1,29 @@
+package com.nervousfish.nervousfish.modules.pairing;
+
+import com.nervousfish.nervousfish.service_locator.IServiceLocator;
+import com.nervousfish.nervousfish.service_locator.IServiceLocatorCreator;
+import com.nervousfish.nervousfish.service_locator.ModuleWrapper;
+
+/**
+ * An adapter to the default Android bluetooth library
+ */
+public final class AndroidNFCAdapter implements INFCHandler {
```

Comment:
No,but the Service Locator must have a valid reference to an implementation of the pairing modules. To implement your suggestion I removed these classes and made 3 anonymous classes (from their interfaces) in ServiceLocatorCreator